libs/input/src/index.tsx

```diff
@@ -0,0 +1,74 @@
+import {
+  FormControl,
+  FormErrorMessage,
+  FormHelperText,
+  FormLabel,
+  Input as ChakraInput,
+  InputGroup,
+  InputRightElement,
+} from '@chakra-ui/react';
+import React, { ChangeEvent } from 'react';
+
+import { IconClose } from '@millicast-react/dolbyio-icons';
+
+import { InputProps } from './types';
+
+const Input = ({ disabled, error, label, helper, onChange, placeholder, required, testId, value }: InputProps) => {
+  const handleChange = (event: ChangeEvent<HTMLInputElement>) => {
+    onChange(event.target.value);
+  };
+
+  const handleClear = () => {
+    onChange('');
+  };
+
+  return (
+    <FormControl isDisabled={disabled} isInvalid={!!error} isRequired={required} test-id={testId}>
+      <FormLabel
+        background="dolbyNeutral.800"
+        fontSize="12px"
+        fontWeight="500"
+        left="12px"
+        m="0"
+        p="0 4px"
+        pointerEvents="none"
+        position="absolute"
+        top={0}
+        transform={'translateY(-50%)'}
+        zIndex={2}
+      >
+        {label}
+      </FormLabel>
+      <InputGroup>
+        <ChakraInput
+          _focus={{ borderColor: 'dolbyPurple.400', boxShadow: 'none' }}
```

Comment:
Same thinking as above, we could define these in `dolbyio-theme` or pass them by arguments

Reply:
As above